### PR TITLE
Hero Section

### PR DESCRIPTION
**Pull request by gsngdavid**
Welcome message
Ojembees image
Three bottom cards
Page responsiveness in all screens

### CODE REVIEW

**Review by UmutesiMMA — CHANGES_REQUESTED** [review]
Hi @gsngdavid 

**CHANGES REQUIRED** 🤏🏾🤏🏾

so-close-keegan-michael-key

- Below is the Figma design, When i compare it with yours some words are taking more width. Kindly check the screenshot below this one.

image


- I have highlighted the areas that I feel need to be adjusted slightly. Kindly check out the image below. 

image

- Kindly feel free to reach out to me in case there is something you aren't understanding.
_Happy Coding_ 🎊 🎊

File: build/sections/hero.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+<div class="container w-full relative py-3 bg-hero bg-cover bg-no-repeat flex flex-col gap-y-8 md:flex-row md:justify-between gap-x-12 lg:gap-x-32 -mt-[68px] text-white">
+    <div class="flex items-center gap-x-4">
+        <div>
+            <img width="120" class="max-w-[60px]" src="Images/hand-coins.png" alt="">
+        </div>
+        <div>
+            <h1 class="text-primary-green">Kostengünstig</h1>
+            <p class="md:max-w-[200px]">niedriges Lohnkostenniveau im europäischen Vergleich</p>
+        </div>
```
Comment (AdelineA) [inline build/sections/hero.html:22]: Looks good, but you forgot to use the fonts they provided in Figma, you may check and apply them where needed.

File: build/sections/hero.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    </div>
+    <div class="flex items-center gap-x-4">
+        <div>
+            <img width="120" class="max-w-[60px]" src="Images/handshake.png" alt="">
+        </div>
+        <div>
+            <h1 class="text-primary-green">Talent & Quality</h1>
+            <p class="md:max-w-[180px]">Viele junge, gut ausgebildete und motivierte Menschen</p>
+        </div>
```
Comment (AdelineA) [inline build/sections/hero.html:31]: Same as here, no fonts applied!!

File: build/sections/hero.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    </div>
+    <div class="flex items-center gap-x-4">
+        <div>
+            <img width="120" class="max-w-[60px]" src="Images/map-pin-line.png" alt="">
+        </div>
+        <div>
+            <h1 class="text-primary-green">Location</h1>
+            <p class="md:max-w-[180px]">Rwanda befindet sich in der selben Zeitzone wie die DACH Region</p>
+        </div>
```
Comment (AdelineA) [inline build/sections/hero.html:40]: Same as here too, fonts!

File: build/sections/hero.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
+<div class="container relative pt-5 xl:min-h-[500px]">
+    <div class="mb-8 xl:mb-0 xl:absolute xl:top-[37%] xl:transform xl:-translate-y-1/2">
+        <p class="text-center xl:text-left text-2xl md:text-[50px] text-white font-bold font-raleway">
+            <span>Win-Win:</span> <br>
+            Europas <span class="text-primary-green inline-block md:my-5">Digitalisierung</span> <br>
+            profitiert von Afrikas Talenten 
+        </p>
+        <p class="text-center xl:text-left mt-3 text-white">Erweiteren Sie Ihre Entwicklungsteams mit Ojemba</p>
```
Comment (AdelineA) [inline build/sections/hero.html:8]: Here, you forgot to use the fonts (both size and family).

**Review by AdelineA — CHANGES_REQUESTED** [review]
Your section looks perfect but you may look at the following:
- You may check provided fonts in Figma and use them where needed.
- On both mobile and md you may increase the padding top so as to make it look perfect.

Below is a screenshot of a small thing you may check:
1. You can add small padding to avoid this section not starting at the same line as the above contents.
david

Here is the Figma you may  check on it:
figma

**Review by UmutesiMMA — APPROVED** [review]
Well done🤝🏾

**Review by AdelineA — APPROVED** [review]
Job well done!

**Pull request merged**